### ISSUE TITLE
Include Sirius in Debian

### ISSUE THREAD

**Issue by irl** [claritylab/lucida#81]
Debian bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=782763

So my normal checklist is:
- [ ] license terms
- [ ] looking for copies of convienience code
- [ ] ensuring all dependencies are packaged already (packaging if not)
- [ ] make sure build can happen in a buildd environment (offline, only using packages from official repos)

@jhauswald has confirmed that the project is still actively maintained in #77. Issues were raised by @mseeber in #37 regarding the current installation process.

As I investigate I will keep this issue updated, creating new issues and/or PRs for any problems found.

**Comment by posophe** [comment]
Hey ! I'm packaging Sirius for openSUSE, then I'm interesting too by this bug report. Just a question about OpenEphyra. I have been able to find a new project with updated stuff into but it's now clean that OpenEphyra is unmaintained yet. Do you plan to move on or maintain it yourselves ?

Regards.
Benjamin

**Comment by akuckartz** [comment]
:+1:

**Comment by posophe** [comment]
I have found a possible alternative : qanus. I don't know exactly how relevant it could be but it's a way to look at.

**Comment by posophe** [comment]
Or yodaqa. Not stable enough to be used on a production system but maybe it's interesting to keep an eye on it.

**Comment by saijel** [comment]
Hi @irl, please open a new issue if this problem still persists. Thanks!

**Comment by irl** [comment]
@saijel Please reopen this bug, this is actively tracked from the Debian BTS and is *not* yet fixed.

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=782763

It is currently blocked by the inclusion of kaldi in Debian (Debian#808427).

I am no longer the contact for this, I didn't have the time, the new contact is Kishan Raval <thekishanraval@gmail.com>.

You should contact this new person if there is renewed activity in the project, we are friendly people and we want to work with you.

@posophe maybe also could do with a ping, on the OpenSUSE front.

**Comment by jhauswald** [comment]
@irl thanks for the update, we'll be sure to follow up with Kishan!

**Comment by Alessandro-Barbieri** [comment]
You need to get rid of patented dependencies because these aren't on Debian
